core/embed/prodtest/main.c

```diff
@@ -421,8 +437,8 @@ static void test_touch_custom(const char *args) {
 
   uint32_t ticks_start = hal_ticks_ms();
 
-  display_clear();
-  display_bar(x, y, width, height, 0xFFFF);
+  gfx_clear();
+  gfx_draw_bar(gfx_rect_wh(x, y, width, height), 0xFFFF);
```

Comment:
and here

Reply:
fixed, see 989fffebf2196106384103bd7800dfdcbb7a54b9